src/main.zig

```diff
@@ -6,6 +6,38 @@ pub const Easy = @import("easy.zig");
 pub usingnamespace Easy;
 pub usingnamespace @import("c.zig");
 
+pub var CaBundle: ?[]const u8 = null;
```

Comment:
Could we avoid this global variable?

I would like to init ca bundle when init `Easy` instance, just like `timeout_ms`, `default_user_agent`.

Reply:
That might be the better way to go. Then you can simply create new Easy instance if you want refreshed ca bundle.

Reply:
@Cloudef Hi, do you still have time to push this further?

I can help if you are busying.

Reply:
Yes I have time today. From tomorrow maybe not so much :slightly_smiling_face: